src/app/about/About.module.css

```diff
@@ -4,6 +4,54 @@
     text-align: center;
 }
 
+/* CSS Modules recommends camelCase
+See: https://github.com/css-modules/css-modules#naming */
```

Comment:
[Nit/Please Change]: We don't need this comment here. While it's helpful information; it applies to CSS modules _everywhere_ throughout the project. The best way to handle this would be in some type of general doc/project info page, but IMO this is also a core piece of CSS modules to learn and reasonably fair to consider as implicit knowledge.